### ISSUE TITLE
error: _chalk(...).default.warn is not a function

### ISSUE THREAD

**Issue by dmelo** [expo/expo-cli#1177]
## Description

When I try to eject a project, I get an error with chalk.

## Expected Behavior

I expected to see a success message at the end.

## Observed Behavior

Here is the full output from expo-cli eject:

```
Warning! Your git working tree is dirty.
It's recommended to commit all your changes before proceeding,
so you can revert the changes made by this command if necessary.

? How would you like to eject your app?
  Read more: https://docs.expo.io/versions/latest/expokit/eject/ Bare
Writing app.json...
Wrote to app.json, please update it manually in the future.
Successfully copied template native code.
Updating your package.json...
_chalk(...).default.warn is not a function
TypeError: _chalk(...).default.warn is not a function
    at ejectToBareAsync (/expo-cli@3.4.1/src/commands/eject/Eject.js:216:15)

```

## Environment

```
  Expo CLI 3.4.1 environment info:
    System:
      OS: Linux 5.2 Fedora 29 (Twenty Nine) 29 (Twenty Nine)
      Shell: 4.4.23 - /bin/bash
    Binaries:
      Node: 10.16.3 - /opt/nodejs/bin/node
      Yarn: 1.19.1 - /usr/bin/yarn
      npm: 6.12.1 - /opt/nodejs/bin/npm
    npmPackages:
      expo: ^34.0.1 => 34.0.4 
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.10 => 0.59.10 
      react-navigation: 3.11.1 => 3.11.1 
    npmGlobalPackages:
      expo-cli: 3.4.1
```

**Comment by dmelo** [comment]
Btw. Looks related to #1172

**Comment by ackratos** [comment]
> I had this same issue (and it was unrelated to #1172). I resolved by removing a `postinstall` script that I had in my package.json. For some reason, when `expo eject` adds its `"postinstall": "jetify"` script to package.json, it throws a breaking _chalk error if there's already a `postinstall` script listed.

this resolved my problem. Thx

**Comment by fson** [comment]
Thanks for these additional details @aaj3f! It turns out the logging code running after modifying the `postinstall` had an error and cause the crash.

This has already been fixed by #1124 and the fix will be included in the next Expo CLI release.
